### ISSUE TITLE
5,00 parses as 500 for decimal format #,##0.### (version 1.1.1)

### ISSUE THREAD

**Issue by JorrisBesselink** [jquery/globalize#644]
While testing globalized validation, found out that 5,00 (dutch 5) validates for the en-US culture that uses #,##0.### as the decimal number format. Debugging shows the parser incorrectly returns the string value "5,00" as 500 thus validating the value as a number where it should not.

Thanks.

**Comment by rxaviers** [comment]
Thanks for filing this issue. The fix for such issue depends on #353.

**Comment by rxaviers** [comment]
Closing this in favor of the former.
